DDS.mwc

```diff
@@ -12,6 +12,7 @@ workspace {
     ACE_wrappers
     // For the case where the user clones git repository
     ATCD
+    ACE_TAO
```

Comment:
Looks like this didn't get updated when the ATCD git repo name changed.  I think we can replace the ATCD line.